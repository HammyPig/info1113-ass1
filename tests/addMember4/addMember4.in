ADD MEMBER fjdio$*W(jkel!!!)A
add member AAAAAAAAAAAAAAAAA HELP MEEEEEEEEEEEEE
aDD mEMBer hello helloe hleLOHE HELOELHO LHEOLHELLO?
MEMBER 100000
MEMBER 100001
MEMBER 100002