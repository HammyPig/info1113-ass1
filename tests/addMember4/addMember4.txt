member name is preserved as entered