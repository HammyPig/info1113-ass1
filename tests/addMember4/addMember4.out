user: Success.

user: Success.

user: Success.

user: 100000: fjdio$*W(jkel!!!)A

user: 100001: AAAAAAAAAAAAAAAAA HELP MEEEEEEEEEEEEE

user: 100002: hello helloe hleLOHE HELOELHO LHEOLHELLO?

user: Ending Library process.
